control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 	"crypto/md5"
 	"fmt"
-
+	cpoutil "github.com/openshift/hypershift/support/util"
```

Comment:
Nit: Can we please keep the existing grouping intact and add this to the group below where all the other hypershift imports are?

Reply:
done